timely_beliefs/beliefs/classes.py

```diff
@@ -1498,3 +1529,6 @@ def downsample_beliefs_data_frame(
         ],
         axis=1,
     ).set_index([belief_timing_col, "source", "cumulative_probability"], append=True)
+
+
+Belief = TimedBelief
```

Comment:
It's true that this doesn't really add functionality. I just added it so that examples in the documentation become a bit shorter.

If we decide **for** it, I would argue to do at least the same for `Source = BeliefSource`, as one of our three custom defined core objects `Sensor`, `Source` and `Belief`. Would it make sense to also have `DataFrame = BeliefsDataFrame`? Although at first glance that seems confusing, it's type would still be BeliefsDataFrame and it would arguably look good in examples. Our first example in the main readme would then become:

```
>>> import timely_beliefs as tb
>>> df = tb.DataFrame([tb.Belief(tb.Sensor("Indoor temperature", "°C"), tb.Source("Thermometer"), 21, event_time="2000-03-05 11:00Z", belief_horizon="0H")])
```

This may be a slippery slope into renaming a lot of things, though. But I would argue not to go further than this. For example, no shorthands for other classes like `[...]DBMixin` classes or other named variables like `belief_horizon`.

Reply:
For clarity, the renaming of "value" to "event_value" in the TimedBelief initialization in this PR is not an attempt to allow both a shorthand and the full variable name. Rather, that is an attempt to migrate away from inconsistent variable naming. But instead of immediately replacing the variable name, I wanted to try out writing a user warning for an upcoming deprecation.

Reply:
Yes, a deprecation across two months or so sounds like the right approach. Even though there are probably no users we don't talk to in person right now.

Reply:
I can't decide pro-shorthand right now. It seems like the gains (short examples) are small, while the potential downsides (confusion, mixing with other libraries' namespaces) seem potentially large.

Maybe postpone this specific discussion? It's larger than this PR.